tests/unit/test_published_clusters.py

```diff
@@ -69,6 +69,17 @@ def test_refresh_ids(self):
         self.assertEqual(op.resource_id, self._operations_json["id"])
         self.assertTrue(op.succeeded())
 
+    @responses.activate
+    def test_refresh_stats(self):
```

Comment:
could this be consolidated into one line?

Reply:
lines 74 and 75

Reply:
Yes, it would just be an extremely long line. I think I'll actually move the base URL to be a class variable since it's used in other tests as well.